src/js/pages/Sync/index.js

```diff
@@ -137,9 +141,23 @@ export default class Sync extends React.PureComponent {
           type="text"
           placeholder="Enter code. Eg. ABC-XYZ"
           pattern="[A-Z,a-z]{3}-[A-Z,a-z]{3}"
+          onKeyUp={e => {
+            const typedValue = e.currentTarget.value;
+            const typedChar = e.key;
+            const parsedValue = typedValue.match('^[A-Z,a-z]{3}');
+            const d = parsedValue ? parsedValue[0] === typedValue : false;
+            if (d && typedChar !== 'Backspace') {
+              e.currentTarget.value = typedValue + '-';
+            }
+          }}
         />
         <button className="sync-form--button">Connect</button>
       </form>
+      {getCode() ? (
+        <button className="reconnect-btn hollow button secondary"
+          onClick={() => { onSubmit(getCode()) }}>Reconnect to {getCode()}</button>
```

Comment:
not using prettier?

Reply:
I am using prettier. Will check this file again.

Reply:
@bogas04 Tried again, prettier doesn't change any indentation in this file. It looks all good to it.